docs/docsite/rst/dev_guide/developing_modules_documenting.rst

```diff
@@ -33,23 +33,28 @@ All modules must have the following sections defined in this order:
 ANSIBLE_METADATA Block
 ----------------------
 
-ANSIBLE_METADATA contains information about the module for use by other tools. At the moment, it informs other tools which type of maintainer the module has and to what degree users can rely on a module's behaviour remaining the same over time.
+``ANSIBLE_METADATA`` contains information about the module for use by other tools. At the moment, it informs other tools which type of maintainer the module has and to what degree users can rely on a module's behaviour remaining the same over time.
 
 For new modules, the following block can be simply added into your module
 
 .. code-block:: python
 
-   ANSIBLE_METADATA = {'status': ['preview'],
-                       'supported_by': 'community',
-                       'version': '1.0'}
+   ANSIBLE_METADATA = {'metadata_version': '1.0',
+                       'status': ['preview'],
+                       'supported_by': 'community'}
 
-.. warning:: version field
+.. warning:: metadata_version field
 
-   This is the version of the ``ANSIBLE_METADATA`` schema, *not* the version of the module.
+   * This is the version of the ``ANSIBLE_METADATA`` schema, *not* the version of the module.
+   * Promoting a module's ``status`` or ``supported_by`` status should only be done by members of the Ansible Core Team.
```

Comment:
The note about `status` and `supported_by` doesn't make sense under the "metadata_version field" heading. Perhaps the heading should just be a general warning instead?